tier2/hooks/post_gen_project.py

```diff
@@ -30,8 +31,52 @@ def addTopic():
     ]
     subprocess.call(gh_cli_command)
 
+def addMaintainer():
+    maintainers = []
+    add_maintainer = True
+    while add_maintainer:
+        maintainer = {}
+        maintainer["role"] = input("Maintainer's Role (Reviewer, Approver, Maintainer): ").strip()
+        maintainer["name"] = input("Maintainer's Name: ").strip()
+        maintainer["github_username"] = input("Maintainer's GitHub Username: ").strip()
+        maintainer["affiliation"] = input("Maintainer's Affiliation (DSAC, CCSQ, CMMI, etc...): ").strip()
+        maintainers.append(maintainer)
+
+        while True:
+            add_maintainer_input = input("Would you like to add another maintainer? (yes/no): ").strip().lower()
```

Comment:
It might be good to have a default behavior for this sort of prompt. In a lot of scripts when a prompt like this comes up it will have something like this at the end: `[y/N]`. This means that there is a default behavior to this prompt when the user hits enter, denoted by the capitalized letter. So in this example if the user hit enter the default response from the script is to treat that as a 'No'. 

To make this change you would just have to allow an empty response and decide on a default behavior for that response. For example if the default is treated as a 'Yes' you would change the prompt to `"Would you like to add another maintainer? [Y/n]: "`.